samples/drivers/video/tcpserversink/src/main.c

```diff
@@ -131,7 +131,10 @@ int configure_encoder(void)
 	}
 
 	buffer->type = VIDEO_BUF_TYPE_OUTPUT;
-	video_enqueue(encoder_dev, buffer);
+	if (video_enqueue(encoder_dev, buffer)) {
+		LOG_ERR("Unable to enqueue encoder output buf");
+		return -1;
+	}
```

Comment:
How about contributing to samples transition to Zephyr-like coding style for new code, so that the original error code can be returned?

```c
	ret = (video_enqueue(encoder_dev, buffer);
	if (ret < 0) {
		LOG_ERR("Unable to enqueue encoder output buf");
		return ret;
	}
```